### ISSUE TITLE
The installation instructions 

### ISSUE THREAD

**Issue by DrBendiM** [CDU-data-science-team/nottshcOpenData#3]
Requires a change from:

remotes::install_github("CDU-data-science-team/nottshcData")

to

remotes::install_github("CDU-data-science-team/nottshcOpenData")

**Comment by ChrisBeeley** [comment]
That's what I'm seeing here? https://github.com/CDU-data-science-team/nottshcOpenData/blob/b77eb152f831c66eaaefeb0654333e9fb4eda360/README.md?plain=1#L17

Or am I missing something?

**Comment by DrBendiM** [comment]
The name of the GitHub repository was for the trust version not the open version. BenOn 27 Sep 2023, at 18:50, Chris Beeley ***@***.***> wrote:﻿
That's what I'm seeing here? https://github.com/CDU-data-science-team/nottshcOpenData/blob/b77eb152f831c66eaaefeb0654333e9fb4eda360/README.md?plain=1#L17
Or am I missing something?

**Comment by ChrisBeeley** [comment]
Sorry, I'm not understanding. This is `nottshcOpenData`, so installation is `remotes::install_github("CDU-data-science-team/nottshcOpenData")`, right?

**Comment by DrBendiM** [comment]
Yes but it was nottshcData originally BenOn 27 Sep 2023, at 21:09, Chris Beeley ***@***.***> wrote:﻿
Sorry, I'm not understanding. This is nottshcOpenData, so installation is remotes::install_github("CDU-data-science-team/nottshcOpenData"), right?

**Comment by ChrisBeeley** [comment]
Log in and view it here https://github.com/CDU-data-science-team/nottshcData
